RJL.HW4.OOP.Classes.Task2/Program.cs

```diff
@@ -38,31 +47,31 @@ static void Main(string[] args)
                 workers[i] = new Worker("SeniourWorkerName_" + indexSenName, "master");
                 indexSenName++;
             }
-            Console.WriteLine("-----------------------------------------------------");
-            Console.WriteLine("Team consists of:");
-            for (int i = 0; i < sumCountWorkers; i++)
-            {
-                Console.WriteLine(i+1+".  "+workers[i].Name+"   "+ workers[i].Experience);
-            }
-            Console.WriteLine("-----------------------------------------------------");
-            Console.WriteLine("Assembly of agregat has been started");
-            Console.WriteLine("-----------------------------------------------------");
-            int tempCountDetails = 0;
-            while (tempCountDetails < Agregat1.AgregatCountDetails) 
+            return workers;
+        }
+        static void GetAssemledAgregat(Agregat agregate, Worker[] workersTeam)
+        {
+            while (agregate.CurrentAssembledDetails < agregate.GeneralCountDetails)
             {
-                for (int i = 0; i < sumCountWorkers; i++)
+                for (int i = 0; i < workersTeam.Length; i++)
                 {
-                    tempCountDetails = workers[i].GetAgregatAssembly(tempCountDetails, Agregat1.AgregatCountDetails);
-                    if (tempCountDetails ==sumCountWorkers)
+                    agregate.CurrentAssembledDetails = workersTeam[i].GetAgregatAssembly(agregate.CurrentAssembledDetails, agregate.GeneralCountDetails);
```

Comment:
I mentioned last time about this, write: `workersTeam[i].DoWorkWithAggregate(aggregate);` instead of  `agregate.CurrentAssembledDetails = workersTeam[i].GetAgregatAssembly(agregate.CurrentAssembledDetails, agregate.GeneralCountDetails);`